components/atoms/BorrowButton.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react";
 import { useNavigation } from '@react-navigation/native';
 import {Button} from "./Button";
+import { StyleSheet } from "react-native";
 
 export const BorrowButton = () => {
     const navigation = useNavigation();
-    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} />
+    return <Button title={"Borrow"} onPress={() => navigation.navigate("Borrow")} style={styles.button} />
 };
+
+const styles = StyleSheet.create({
+    button: {
+        borderTopLeftRadius: 0,
+        borderBottomLeftRadius: 0,
```

Comment:
Can we remove them?

Reply:
Actually we can remove the borderRadius with value 20, but we have to keep these ones.